### DISCUSSION TITLE
Tool versions

### DISCUSSION THREAD

**Discussion by matty0ung** [Brewtarget/brewtarget#559] [Q&A] [answered]
I'm sure this has been discussed elsewhere, but I couldn't immediately find it.  Do we definitely need version 3.18 of CMake?  If we do, it's fine, but it's just that Ubuntu 20.04 only gives you 3.16.3 in the standard packages, so I'd have to download manually (which I can do if CMake 3.16 won't cut it, but just wanted to check first).

**Comment by xzfantom** [comment, marked as answer]
In GitHub Actions it works with no additional cmake installation. I just found that they have preinstalled 3.20 version, so there is no error.
Main needed feature is GET_RUNTIME_DEPENDENCIES and it appeared in CMake 3.16, so probably 3.16 is enough.

**Comment by matty0ung** [comment]
We're on newer versions of Ubuntu now, so archiving this.
